skin.json

```diff
@@ -291,7 +291,6 @@
       {"name":"live", "location":"controlBar", "whenDoesNotFit":"keep", "minWidth":45},
       {"name":"timeDuration", "location":"controlBar", "whenDoesNotFit":"drop", "minWidth":145 },
       {"name":"flexibleSpace", "location":"controlBar", "whenDoesNotFit":"keep", "minWidth":1 },
-      {"name":"chromecast", "location": "controlBar", "whenDoesNotFit":"moveToMoreOptions", "minWidth":45 },
```

Comment:
Are you sure the mobile teams do not use this line?

Reply:
Hope they don't, since it's a key of buttons.desktopContent array.

Reply:
Can we confirm with them?

Reply:
Just confirmed